static/src/javascripts/projects/common/modules/tailor/fetch-data.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import fetchJson from 'lib/fetch-json';
+import { getCookie } from 'lib/cookies';
+import { local as localStorage } from 'lib/storage';
+import reportError from 'lib/report-error';
+import config from 'lib/config';
+
+const BROWSER_ID = getCookie('bwid');
+const URLS = {
+    suggestions: 'https://tailor.guardianapis.com/suggestions?browserId=',
+};
+
+const getURL = (type: string, queryParams?: Object): ?string => {
+    if (!BROWSER_ID || !URLS[type]) {
+        return;
+    }
+
+    let baseURL = URLS[type] + BROWSER_ID;
+
+    if (queryParams) {
+        Object.keys(queryParams).forEach(key => {
+            if (queryParams) {
```

Comment:
It's a [flow issue](https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoALgTwA4FMzpgC8YA3qgJADWumAXGAM7oBOAlgHYDmqAvqgGM4HZmCEd0uAB7oAMnEaNiYABSM4AW1wB5AEYArAPwN0ASgYA3OGwAmxAHxlUYF2DZRVAQnVa9+006uQWAsuOgAriwcANzOrvxxLuKiCHAsVEokPjoGANoA5DSY+QC6sUHJhLqhAIYZykUOTJo5+rlFZXzRQA) - or feature. Who know? :/